cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ModelUtils.java

```diff
@@ -166,4 +169,30 @@ public static double expectedUtilizationFor(Resource resource, AggregatedMetricV
     }
     return max(result, 0.0);
   }
+
+  /**
+   * Check the consistency between a Partition object and a PartitionInfo object
+   * @param partition the Partition object that contains partition info
+   * @param partitionInfo the PartitionInfo object that contains partition info
+   * @return true if both objects contains the same partition info
```

Comment:
Nit: Can we use proper tags -- e.g. `{@code true}`, and `{@link }` for Partition and PartitionInfo.